qiskit_ibm_runtime/utils/utils.py

```diff
@@ -118,6 +118,16 @@ def get_runtime_api_base_url(url: str, instance: str) -> str:
     return api_host
 
 
+def _is_experimental_runtime_url(url: str):
+    """Checks if the provided url points to an experimental runtime cluster. This type of URLs is used for internal
+    development purposes only.
+
+    Args:
+        url: The URL.
+    """
+    return isinstance(url, str) and "experimental" in url and url.endswith(".cloud")
```

Comment:
I don't like the dependency on the internal cluster name here but cannot think of a better solution that does not require a bigger rework in the library and changes to the existing public interface.